OpenRCT2.xcodeproj/project.pbxproj

```diff
@@ -4498,6 +4498,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				CLANG_ENABLE_OBJC_WEAK = YES;
+				MACOSX_DEPLOYMENT_TARGET = 10.7;
```

Comment:
As we have established, this is an unrealistic target. Could you change all of these to read 10.13 instead?

(No need to amend the commit, we can squash the lot on merge.)

Reply:
```suggestion
				MACOSX_DEPLOYMENT_TARGET = 10.13;
```

You are us can also accept this suggestion which is a quick way to commit the change.

Reply:
It's not just one line, though. I didn't feel like making six suggestions.